doc/changelog.rst

```diff
@@ -9,6 +9,16 @@ Unrealeased
   When releasing next version, the "Unreleased" header will be replaced
   with appropriate version header and this help text will be removed.
 
+Core
+~~~~
+
+- Add support for Django 1.11 and for now on Shuup is installed over Django 1.11
+by default. There is still unofficial support for Django 1.8 and 1.9 which
+means that after installing Shuup you can downgrade your Django and Django
+polymorphic versions. We will also still run tests for these old versions
+while adding new features to Shuup. See tox.ini for information about
+downgrades required to run Shuup with old Django.
```

Comment:
Shouldn't this be in "General changes" section rather than core? It's concerning all of the Shuup apps, isn't it?

Reply:
Now OK